internal/commands/app/errors.go

```diff
@@ -1,7 +1,14 @@
 package app
 
-type errProjectExists struct{}
+type errProjectExists struct {
+	details string
```

Comment:
any objections to making this `path string`, and then using
```go
var suffix string
if err.path != "" {
  suffix = " at "+err.path
}
return "a project already exists" + suffix
```
for the error string?